atomic_reactor/plugins/exit_verify_media_types.py

```diff
@@ -57,7 +60,20 @@ def run(self):
             insecure = registry.get('insecure', False)
             secret = registry.get('secret', None)
 
-            digests = get_manifest_digests(pullspec, registry_name, insecure, secret,
+            if PLUGIN_FETCH_SOURCES_KEY in self.workflow.prebuild_results:
+                # For source containers, limit the expected media
+                # types.  This can help to avoid issues with tooling
+                # that is unable to deal with the number of layers in
+                # these images.
+                src_config = get_source_container(self.workflow, fallback={})
+                limit_media_types = src_config.get('limit_media_types')
+                if limit_media_types is not None:
+                    expected_media_types.intersection_update(set(limit_media_types))
```

Comment:
shouldn't we just  ? : 
expected_media_types = set(limit_media_types)  

because we don't usually set expected_media_types 
so expected_media_types would end up empty

Reply:
Ah, good catch.